packages/mobile/src/components/components.ts

```diff
@@ -1,8 +1,12 @@
-export { View } from 'react-native';
+export { View, StatusBar, Pressable, ScrollView } from 'react-native';
 export { SafeAreaView } from 'react-native-safe-area-context';
 export { ScreenWrapper } from './screen-wrapper/screen-wrapper';
 export { Button } from './button/button';
 export { Input } from './input/input';
 export { Text } from './text/text';
 export * from './icons/icons';
 export { Spinner } from './spinner/spinner';
+export { default as LinearGradient } from 'react-native-linear-gradient';
```

Comment:
move import from library to the top for keeping the structure

Reply:
corrected